From e6ac7197b2d48bce8cc945136a6723f0ab9c9a29 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Fri, 3 Jul 2015 14:12:25 +0800
Subject: [PATCH 1501/1594] MLK-11123-28 ARM: dts: disable otg features for
 imx6sx-sdb board

commit 802071ec3099699eaeb4ca0ad80c51bf6e63d297 from
git://git.freescale.com/imx/linux-2.6-imx.git

Disable all otg features for otg1 port of imx6sx-sdb board, then
the otg port can only do role swtich by ID pin state.

Signed-off-by: Li Jun <jun.li@freescale.com>
(cherry picked from commit e8d32c2cc5daab7e22f6190a02c88981aa14df7e)
---
 arch/arm/boot/dts/imx6sx-sdb.dtsi |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dtsi b/arch/arm/boot/dts/imx6sx-sdb.dtsi
index d3f59e6..16b47bd 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dtsi
+++ b/arch/arm/boot/dts/imx6sx-sdb.dtsi
@@ -455,6 +455,9 @@
 	vbus-supply = <&reg_usb_otg1_vbus>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usb_otg1_id>;
+	srp-disable;
+	hnp-disable;
+	adp-disable;
 	status = "okay";
 };
 
-- 
1.7.5.4

